src/helpers/utils.js

```diff
@@ -0,0 +1,3 @@
+exports.generateCode = function () {
+  return Math.floor(Math.random() * 9000) + 1000;
+};

```

Comment:
chnage to :
exports.generateCode = function (len=4) {
    var code = ""
    for (var i = 0; i < len; i++) {
        code = code + Math.floor(Math.random() * 10)
    }
    return code
}

Reply:
done